.eslintrc.json

```diff
@@ -26,7 +26,7 @@
   },
   "rules": {
     "prettier/prettier": "error",
-    "indent": ["error", 2],
+    "indent": ["error", 2, { "SwitchCase": 1 }],
```

Comment:
This is the existing `eslint-config` rule: https://github.com/zendeskgarden/eslint-config/blob/master/rules/stylistic-issues.js#L58. Can just be deleted.

Reply:
I'm not sure what's going on with our custom config, but it seems prettier is fighting the config on this one and can't seem to align on anything. 

Lets meet offline to walk through this.

Reply:
If there are plugin rules unintentionally overriding our config, then I think the `extends` section needs to be reordered, like so:

```json
{
  "extends": [
    "prettier",
    "@zendeskgarden",
    "@zendeskgarden/eslint-config/plugins/notice.js",
    "plugin:react/recommended",
    "plugin:jest/recommended",
    "plugin:jsx-a11y/recommended"
  ]
}
```

based on https://eslint.org/docs/user-guide/configuring#extending-configuration-files